python/cucim/MANIFEST.in

```
@@ -22,4 +22,6 @@ include tox.ini .travis.yml .appveyor.yml .readthedocs.yml
 include versioneer.py
 include src/cucim/clara/*.so*
 
+recursive-include src/cucim *.py *.pyi *.cu *.h *.npy *.txt *.in *.md
```

FWIW `MANIFEST.in` is automatically included. So if that is the only reason for `.in`, we can drop `.in`

```suggestion
recursive-include src/cucim *.py *.pyi *.cu *.h *.npy *.txt *.md
```